cube.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module defines a Cube class, and prints a list of 25
+randomly chosen faces of the cube.
+"""
+
+import random
+
+
+# Number of moves in scramble algorithm.
+K = 25
```

Comment:
You should pass this to whatever method is returning a scramble.

Reply:
This should get passed that way even if I want `K` to always be 25?